### PR TITLE
CONCF 633 | Forum thread subject line is indented, not using full width available

### PR DESCRIPTION
**Pull request by tomasznapieralski**

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
@hakubo @rogatty

**Comment by rogatty** [comment]
LGTM

File: extensions/wikia/Forum/css/Forum.scss [hunk 1]
```diff
@@ -287,14 +287,16 @@
...
-	padding-right: 260px;
+	padding-right: 0;
 	position: relative;
 
+	h1 {
+		margin: 0;
+		word-break: normal;
+	}
+
 	.header-right {
```
Comment (hakubo) [inline extensions/wikia/Forum/css/Forum.scss]: .WikiaPage .WikiaPageHeader this looks like a generic class that touches every page
are we sure that Forum.css is not loaded on other pages? :)

Comment (rogatty) [inline extensions/wikia/Forum/css/Forum.scss]: /cc @bognix @Warkot

Comment (bognix) [inline extensions/wikia/Forum/css/Forum.scss]: LGTM as a quick fix. Although the best idea would be to apply markup and styles from article so it becomes "social share icons" friendly.

Comment (tomasznapieralski) [inline extensions/wikia/Forum/css/Forum.scss]: https://wikia-inc.atlassian.net/browse/CONCF-664

Comment (bognix) [inline extensions/wikia/Forum/css/Forum.scss]: LGTM

**Pull request merged**